### ISSUE TITLE
Schema validation error for devfile

### ISSUE THREAD

**Issue by dneary** [che-incubator/chectl#182]
I tried to open the Chectl project with the devfile.yaml, and it failed at schema validation:
```
Loading factory
Error occurred during creation a workspace from devfile located at 
`https://raw.githubusercontent.com/che-incubator/chectl/master/devfile.yaml`.
Cause: Devfile schema validation failed.
Error:
(/apiVersion):The object must not have a property whose name is "apiVersion".
(/metadata):The object must not have a property whose name is "metadata".
The object must have a property whose name is "specVersion".
The object must have a property whose name is "name".
```

**Comment by metlos** [comment]
The devfiles with `apiVersion` only work with Che 7 RC 2 and newer. What version is your Che server?

**Comment by l0rd** [comment]
@metlos I suspect @dneary was using che.openshift.io that has not been updated to RC2

@dneary I am closing this issue because your problem will be solved as soon as Che 7.0.0 will be deployed on che.openshift.io. In the meantime you can use [this devfile]( https://gist.githubusercontent.com/l0rd/85a5680b933aba045654904a6aee6ef7/raw/bff6b582546559a8ff60052d2fe521debdf6f634/devfile.yaml). Or just [click here](https://che.openshift.io/dashboard/f?url=https://gist.githubusercontent.com/l0rd/85a5680b933aba045654904a6aee6ef7/raw/bff6b582546559a8ff60052d2fe521debdf6f634/devfile.yaml)

**Comment by dneary** [comment]
Yes, I was using che.openshift.io - I would have expected that devfiles would be compatible with our hosted Che, or that we would already have started to use apiVersion increments to allow servers to say what they support and fail more elegantly than a schema validation error
